main.rb

```diff
@@ -0,0 +1,10 @@
+require './app'
+
+def main
```

Comment:
- [ ] According to the project requirements, the school library console app should behave in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) it was provided for them. Actually, your console does not work as expected as you can see on the screen video 👇🏻 

[Screencast from 08-05-2022 02:19:07 PM.webm](https://user-images.githubusercontent.com/79058364/183076536-9fc5de46-a2c1-4adc-ace0-67f50d9b5880.webm)

- Kindly address this!!